runtime/kusama/src/lib.rs

```diff
@@ -1301,3 +1301,75 @@ sp_api::impl_runtime_apis! {
 		}
 	}
 }
+
+#[cfg(test)]
+mod test_fees {
+	use super::*;
+	use frame_support::weights::WeightToFeePolynomial;
+	use frame_support::storage::StorageValue;
+	use sp_runtime::FixedPointNumber;
+	use frame_support::weights::GetDispatchInfo;
+	use codec::Encode;
+	use pallet_transaction_payment::Multiplier;
+	use separator::Separatable;
+
+
+	#[test]
+	#[ignore]
```

Comment:
What is the purpose of these ignored tests?

Reply:
The other aspect of the PR that we decided to address later: it help print some stat about the fee stuff.